pkg/cloudprovider/providers/aws/retry_handler.go

```diff
@@ -92,6 +92,7 @@ func (c *CrossRequestRetryDelay) AfterRetry(r *request.Request) {
 	}
 	if awsError.Code() == "RequestLimitExceeded" {
 		c.backoff.ReportError()
+		recordAWSThrottlesMetric()
```

Comment:
Yeah the other metrics were designed to be uniform across cloudproviders and hence selected operation names were decided as such. Since this will be AWS specific metrics I am fine keeping the label values to be `DescribeInstance` .

Reply:
+1 for having a label for the operation on the throttle.

Reply:
Done. Thanks for taking a look.